Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -123,6 +132,27 @@ extension SettingsDictionary {
     public func swiftOptimizeObjectLifetimes(_ enabled: Bool) -> SettingsDictionary {
         merging(["SWIFT_OPTIMIZE_OBJECT_LIFETIME": SettingValue(enabled)])
     }
+    
+    // MARK: - Swift Compiler - General
+
+    /// Sets `"SWIFT_OBJC_BRIDGING_HEADER"` to `path`
+    public func swiftObjcBridingHeaderPath(_ path: String) -> SettingsDictionary {
+        merging(["SWIFT_OBJC_BRIDGING_HEADER": SettingValue(path)])
+    }
```

Comment:
I believe there can only be one bridging header per target - as such `merging` may cause multiple headers to be resolved in a setting.

Reply:
I agree, any suggestions on how to proceed?

Reply:
`SettingsDictionary` is a [type alias to a regular dictionary](https://github.com/tuist/tuist/blob/main/Sources/ProjectDescription/Settings.swift#L3) as such I believe a simple subscript assignment should do the trick

```swift
self["SWIFT_OBJC_BRIDGING_HEADER"] = .string(path)
```

Reply:
Done @kwridan, can you please validate it?